projects/plugins/jetpack/extensions/blocks/premium-content/block.json

```diff
@@ -50,22 +50,6 @@
 		"isPremiumContentChild": "isPremiumContentChild"
 	},
 	"attributes": {
-		"newPlanName": {
```

Comment:
These attributes are not used anywhere any more. Removing them doesn't break backward compatibility.